css/properties.json

```diff
@@ -4094,6 +4094,21 @@
     ],
     "status": "standard"
   },
+  "line-height-step": {
+    "syntax": " none | <length> ",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Fonts"
+    ],
+    "initial": "none",
+    "appliesto": "blockContainerElements",
+    "computed": "absoluteLength0ForNone",
+    "order": "perGrammar",
+    "status": "nonstandard"
```

Comment:
My sense is that 'experimental' would be more consistent with what you guys typically do. Changing it now.